dialoptions.go

```diff
@@ -428,6 +428,11 @@ func WithTimeout(d time.Duration) DialOption {
 // returned by f, gRPC checks the error's Temporary() method to decide if it
 // should try to reconnect to the network address.
 //
+// If you want to bypass DNS resolution and connect directly to a specified
+// address, prefix your target address with passthrough:/// when using
+// WithContextDialer. This will cause the passthrough resolver to be used,
+// which avoids any DNS lookups. For example: passthrough:///localhost:50051.
```

Comment:
"you" is too informal for docstrings.  Proposed change:

```suggestion
// Note that gRPC by default performs name resolution on the target passed to NewClient.
// To bypass name resolution and cause the target string to be passed directly to the dialer
// here instead, use the "passthrough" resolver by specifying it in the target string,
// e.g. "passthrough:target".
```

(Let's not suggest `localhost:port` since that should work fine with DNS.  This would be if DNS _doesn't work with_ a target string.)

Reply:
Done